Sources/MultipartFormDataParser/MultipartFormDataParser.swift

```diff
@@ -23,6 +23,9 @@ private extension MultipartFormDataParser {
         while stream.hasBytesAvailable {
             var buffer = [UInt8](repeating: 0, count: 512)
             let count = stream.read(&buffer, maxLength: buffer.count)
+            if count <= 0 {
```

Comment:
```suggestion
            guard count > 0 else {
```

I prefer to use `guard` than `if` on this case.

Reply:
I will merge it briefly, and fix with this suggestion. thx!